BoardConfigPlatform.mk

```diff
@@ -137,6 +140,7 @@ endif
 OVERRIDE_RS_DRIVER := libRSDriver_adreno.so
 
 ### RIL
+DEFAULT_APNS_FILE := $(PLATFORM_PATH)/config/telephony/apns-conf.xml
```

Comment:
Please remove this until LOS fixed their script. See https://github.com/whatawurst/android_device_sony_yoshino-common/issues/46#issuecomment-857421284 for details